docs/issues_catched.md

```diff
@@ -0,0 +1,34 @@
+
+# Issues Precaution catches
+
+Precaution utilizes multiple security static code analysis tools.
+Because of that the amount and types of checks for the different
+programming languages are different.
+The security static code analyzers are language-focused
+and some of the vulnerabilities they find are specific 
+for the concrete programming language.
+However, some of those rules can be grouped into a few groups:
+
+* Hardcoded credentials
+* SQL injections
+* Command injections
+* HTML injections
+* Weak random number generator
+* Imported/used weak cryptographic algorithms
```

Comment:
Why forward slash?

Reply:
Yes, you are right. If you import weak cryptographic algorithms you are going to use them too.